frame/clearing-house/src/math.rs

```diff
@@ -370,7 +370,11 @@ fn multiply_by_rational(mut a: u128, mut b: u128, mut c: u128) -> Result<u128, &
 
 /// Fixed point division with remainder using the underlying integers (`a` and `b`) and the
 /// precision (`acc`) of the fixed point implementation.
-fn div_rem_with_acc(mut a: u128, mut b: u128, mut acc: u128) -> Result<(u128, u128), &'static str> {
+pub fn div_rem_with_acc(
```

Comment:
TBH, I think implementing this function using `sp_core::U256` internally instead of `sp_arithmetic::biguint`, since the latter is very intricate and I can't guarantee that some of the `div` operations work as expected. 

What do you think @Cardosaum @kevin-composable ? Shall I include it in this PR?